source/mobile/mobile-appconfig.rst

```diff
@@ -36,6 +36,10 @@ The following table shows all the configuration options that can be sent from th
 +------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
 | username               | Set the username or email address to use to authenticate against the Mattermost Server.                                                                                                                         | String |               |                  | Yes         | Yes                      |
 +------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
+| useVPN                 | Enable connection to the Mattermost Server to use a per-app VPN or VPN on-demand.                                                                                                                               | String | ``false``     | ``true | false`` | Yes         | No                       |
++------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
```

Comment:
@enahum do we need to add this to any other places in the docs? ie instructions on using a VPN connection?

Reply:
This is the only place we have documentation about this. What are you thinking?

Reply:
Wondering if we should have a section in this doc for setting up VPN? But let's merge this for now and then we can discuss